### DISCUSSION TITLE
Possible to show post content widget twice?

### DISCUSSION THREAD

**Discussion by kripalmen** [elementor/elementor#23845] [Developers Community]
Is there a work a round to add two post content widget to the blog post template?

I want to have one view for desktop and mobile, and in each view, have a post content widget (so two times).
But elementor can only show one post content widget on the frontend.

So any work a rounds?

**Comment by kripalmen** [comment]
Anyone?

**Comment by sreidnoe** [comment]
Are you building a template in theme builder and calling a query for
render?

**Reply by kripalmen** [reply]
No - im just using the elementor widget Blog post - but it is not possible to have it twice on the same page. So it is not possible to make a computer version, and a mobil version.
